src/server/room-controller.js

```diff
@@ -260,6 +262,37 @@ export default class RoomController {
     return res.status(201).end();
   }
 
+  async handlePostRoomMessage(req, res) {
+    const { user } = req;
+    const { roomId } = req.params;
+    const { text, emailNotification } = req.body;
+
+    const room = await this.roomService.getRoomById(roomId);
+    if (room.owner !== user._id) {
```

Comment:
We could use our unified `room-utils` functions here: `isRoomOwner`

Reply:
True, I switch to using it in all such cases in the file. Thanks!
It could make alot of difference if one day the rule of determining ownership changed 😊